nix/pkgs/libspdk/default.nix

```diff
@@ -21,8 +22,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "openebs";
     repo = "spdk";
-    rev = "bef8258c3116d8d427d46fab5985bb080ffb7666";
-    sha256 = "00xsfhlqiwnq3777xg8x6chcqr4nxhhqqmbsq1a2gng2iyhlpxzd";
+    rev = "f09e30d7789aed7f16317c865203ae5d1b9a5d5c";
```

Comment:
There is a newer commit (as of today), should we point it at this version (hash - 0dd3398ace940bb291e18138a2535cf6b1e161b5)?